test/fixtures/e2e/amp-position-observer/scrollbound-animation.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<html ⚡>
+
+<head>
+  <meta charset="utf-8">
+  <title>amp-position-observer</title>
+  <link rel="canonical" href="amps.html">
```

Comment:
Is this necessary?

Reply:
canonical link is required for AMP document.